mongodb_migrations/config.py

```diff
@@ -19,33 +22,42 @@ def __init__(self):
         self._from_ini()
         self._from_console()
 
-        if not self.mongo_database:
-            raise Exception("No database name is provided")
+        if all([self.mongo_url, self.mongo_database]) or not any([self.mongo_url, self.mongo_database]):
```

Comment:
just one minor issue here, since `mongo_url` already contains `host`, `port` and `database`, can we make it to say once `mongo_url` is provided, none of `host`, `port` and `database` can be provided.

Reply:
Fixed

Reply:
just message is updated, but this check is still the same

Reply:
I will need to remove the default values for `host` and `port` in order to update the conditional the way you are asking. I can do that but I just want to be sure that is what you want.

Reply:
Well, after a second thought, I think this one also works now, as `database` can only be provided with `host` and `port`.